terraform/modules/auth_internal_alb/internal_alb.tf

```diff
@@ -34,7 +34,7 @@ resource "aws_security_group_rule" "alb_https_ingress" {
   description       = "HTTPS Ingress"
 }
 
-# The auth albs are shared by keycloak and the auth service so we define the listener here and the rules in each repository
+# The auth albs were shared by keycloak and the auth service so we defined the listener here and the rules in each repository
```

Comment:
Should comments still reference things we don't do anymore even if its past tense?

Reply:
I was unsure about moving the listener definition and wanted to leave a reason as to why it was defined separately to the auth alb rules, agree the comment does feel a bit odd

Reply:
If its still valid for the auth service it may be better to remove the references to keycloak and just leave it as a reference to the auth service.

Reply:
It's probably not worth worrying about moving it into the other repo, but I'd say it's potentially useful to know it's only split up this way for legacy reasons. 

I'd keep the comment, but if you like, it could be rephrased to something like:

The auth service ALB used to be shared with another service that no longer exists, so the rules are defined in the auth service repository while the listener is defined in this repository for legacy reasons.